### PR TITLE
Fix new day stuff

### PR DESCRIPTION
**Pull request by JTawdawd**
- Fixed disabling button if we've already used it today
- Added persistence to music index
- Removed Date checker & local storage manager (check comments)
- Fixed absurd snow div copy paste

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -225,8 +32,28 @@ export default {
...
+    
+    if (this.pageLoadedOn !== new Date().toLocaleDateString()) {
+      this.newDay = true;
+      this.pageLoadedOn = new Date().toLocaleDateString();
+      localStorage.setItem('pageLoadedOn', this.pageLoadedOn);
+    } else {
+      this.newDay = false;
+    }
+
+    this.openedTodaysGift = localStorage.getItem('lastGiftDate') === new Date().toLocaleDateString();
```
Comment (JTawdawd) [inline src/App.vue:56]: This replaces Date Checker

File: src/App.vue [hunk 2]
```diff
@@ -1,222 +1,29 @@
...
+  />
 </template>
 
 <script>
 
 import WheelSpinner from './components/WheelSpinner.vue'
 import MusicPlayer from './components/MusicPlayer.vue'
 
-
+const SNOW_INTENSITY = 200;
```
Comment (JTawdawd) [inline src/App.vue:26]: Control amount of snow here

File: src/components/WheelSpinner.vue [hunk 3]
```diff
@@ -1,79 +1,78 @@
...
           const removedItem = this.wheelArray.splice(chosenOne, 1)[0];
           this.randomNumber -= 1;
           this.removedItems.push(removedItem);
 
-          this.$refs.localStorageManager.saveToLocalStorage('removedItems', this.removedItems);
-          this.$refs.localStorageManager.saveToLocalStorage('wheelArray', this.wheelArray);
+          localStorage.setItem('removedItems', JSON.stringify(this.removedItems));
+          localStorage.setItem('wheelArray', JSON.stringify(this.wheelArray));
+
+          localStorage.setItem('lastGiftDate', new Date().toLocaleDateString());
```
Comment (JTawdawd) [inline src/components/WheelSpinner.vue:69]: No need for wrapper

File: src/components/WheelSpinner.vue [hunk 4]
```diff
@@ -86,36 +85,8 @@ export default {
...
-  computed: {
-    isButtonDisabled: {
-      get() {
-        return localStorage.getItem('isButtonDisabled') === 'true';
-      },
-      set(value) {
-        localStorage.setItem('isButtonDisabled', value);
-      },
-    },
-  },
```
Comment (JTawdawd) [inline src/components/WheelSpinner.vue:105]: The above is handled in app > mounted

File: src/components/WheelSpinner.vue [hunk 5]
```diff
@@ -86,36 +85,8 @@ export default {
...
-    const storedWheelArray = localStorage.getItem('wheelArray');
-    if (storedWheelArray) {
-      this.wheelArray = JSON.parse(storedWheelArray);
-    } else {
-      this.wheelArray = wheelData;
     }
-
-    const storedRemovedItems = localStorage.getItem('removedItems');
-    if (storedRemovedItems) {
-      this.removedItems = JSON.parse(storedRemovedItems);
```
Comment (JTawdawd) [inline src/components/WheelSpinner.vue:116]: All this is now done directly on instantiating data

**Review by MarcyAnth — APPROVED** [review]
thanks guys!

**Pull request merged**